tests/unit/components/form/GitAuthenticationForm.spec.js

```diff
@@ -127,31 +105,10 @@ describe('Test component: GitSettingsForm', () => {
 
       expect(getProjectById).toBeCalled();
       expect(saveProject).toBeCalled();
-      expect(gitAddRemote).toBeCalled();
       expect(wrapper.vm.submitting).toEqual(false);
       expect(wrapper.emitted()['project-git:save']).toBeTruthy();
       expect(emit).toBeCalled();
       expect(Notify.create).toHaveBeenCalledWith(expect.objectContaining({ type: 'positive' }));
     });
-
-    it('should emit a notification on error', async () => {
```

Comment:
I used the previous file `GitSettingsForm.spec.js` and rename it to `GitAuthenticationForm.spec.js` but I don't need to test an error (since I only want to save the data in the localStorage)